book/types_of_data.md

```diff
@@ -362,29 +360,83 @@ You can also create a table as a list of records, JSON-style:
 ╰───┴──────┴──────╯
 ```
 
-:::tip
-Internally, tables are simply **lists of records**. This means that any command which extracts or isolates a specific row of a table will produce a record. For example, `get 0`, when used on a list, extracts the first value. But when used on a table (a list of records), it extracts a record:
+To access a row or a column, you can use the [`get`](/commands/docs/get.md) command or cell path access:
 
 ```nu
-> [{x:12, y:5}, {x:3, y:6}] | get 0
-╭───┬────╮
-│ x │ 12 │
-│ y │ 5  │
-╰───┴────╯
+> [{name: sam, rank: 10}, {name: bob, rank: 7}] | get 1
+╭──────┬─────╮
+│ name │ bob │
+│ rank │ 7   │
+╰──────┴─────╯
+> [{name: sam, rank: 10}, {name: bob, rank: 7}].1
+╭──────┬─────╮
+│ name │ bob │
+│ rank │ 7   │
+╰──────┴─────╯
+> [{name: sam, rank: 10}, {name: bob, rank: 7}] | get name
+╭───┬─────╮
+│ 0 │ sam │
+│ 1 │ bob │
+╰───┴─────╯
+> [{name: sam, rank: 10}, {name: bob, rank: 7}].name
+╭───┬─────╮
+│ 0 │ sam │
+│ 1 │ bob │
+╰───┴─────╯
 ```
 
-This is true regardless of which table syntax you use:
+Note, the records in a table don't necessarily need to have matching keys. If a record lacks a key, the table will lack a value for that column. You can think of the table having a hole.
```

Comment:
This is true but it may be a good place to talk about the difference between these items in tables.
- absence of a value
- empty string
- null value